src/consensus/base.rs

```diff
@@ -1,28 +1,28 @@
 use super::*;
-use crate::{
-    chain::{difficulty::*, protocol_param::param},
-    models::*,
-    state::*,
-};
+use crate::{chain::protocol_param::param, models::*, state::*};
 use anyhow::Context;
 use async_recursion::*;
 use ethereum_types::*;
 use std::time::SystemTime;
 
 #[derive(Debug)]
 pub struct ConsensusEngineBase {
-    chain_config: ChainConfig,
+    chain_id: ChainId,
+    eip1559_block: Option<BlockNumber>,
```

Comment:
Typically we're not using `eipNNN` naming for things in the code. Is there a colloquial name for this? Maybe `london_gas_block`? (as we refer to fork names in some other places)

Reply:
EIP-1559 _is_ the colloquial name for the fee market change. For this reason, I've named this field so, and not `london_block`.